dali/operators/math/expressions/expression_factory_instances/expression_impl_factory.h

```diff
@@ -181,8 +181,8 @@ std::unique_ptr<ExprImplBase> ExprImplFactoryTernaryOp(const ExprFunc &expr) {
 #define IMPLEMENT_OP_FACTORY_GPU_BINARY(OP)                                                 \
   std::unique_ptr<ExprImplBase> OpFactory(arithm_meta<ArithmeticOp::OP, GPUBackend>,        \
                                           const ExprFunc &expr) {                           \
-    return ExprImplFactoryBinOp<ArithmeticOp::OP, GPUBackend, ExprImplGpuTT, ExprImplGpuTC, \
-                                ExprImplGpuCT>(expr);                                       \
+    return ExprImplFactoryBinOp<ArithmeticOp::OP, GPUBackend, ExprImplGpuTT, ExprImplGpuTT, \
+                                ExprImplGpuTT>(expr);                                       \
```

Comment:
This looks to be ahead of this PR, isn't it?

Reply:
Yes, reverting